### PR TITLE
Add documentation for Authorization 2.0 Tech Preview

### PR DESCRIPTION
**Pull request by tdawe**
# Description
Add documentation for Authorization 2.0 Tech Preview

All changes from https://github.com/dell/csm-docs/tree/feature/authorization-tech-preview but updated so that this can be merged into release-1.11.0. After this PR is merged, the branch `authorization-tech-preview` can also be deleted.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1281 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

**Review by KerryKovacevic — APPROVED** [review]

File: content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    ```yaml
+    modules:
+      # Authorization: enable csm-authorization for RBAC
+      - name: authorization
+        # enable: Enable/Disable csm-authorization
+        enabled: true
+        configVersion: v1.10.0
+        components:
+        - name: karavi-authorization-proxy
+          image: dellemc/csm-authorization-sidecar:v1.10.0
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md:106]: should we point to the v2.0.0-alpha image here?

**Review by KerryKovacevic — APPROVED** [review]

File: content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md (hunk 1 above)
Comment (tdawe) [reply content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md]: Fixed

**Review by KerryKovacevic — APPROVED** [review]

File: content/docs/deployment/helm/modules/installation/authorization/_index.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ description: >
 The CSM Authorization karavictl CLI is no longer actively maintained or supported. It will be deprecated in a future release.
 {{% /pageinfo %}}
 
+>> NOTE: Authorization v2.0 Tech Preview is not actively supported through Helm.
```
Comment (gallacher) [inline content/docs/deployment/helm/modules/installation/authorization/_index.md:12]: "actively" can be removed

Comment (tdawe) [reply content/docs/deployment/helm/modules/installation/authorization/_index.md]: Fixed

**Review by KerryKovacevic — APPROVED** [review]

**Review by atye — APPROVED** [review]

**Pull request merged**